nox/manifest.py

```diff
@@ -172,9 +172,20 @@ def filter_by_keywords(self, keywords: str) -> None:
                 session names are checked against.
         """
         self._queue = [
-            x for x in self._queue if keyword_match(keywords, x.signatures + [x.name])
+            x
+            for x in self._queue
+            if keyword_match(keywords, x.signatures + x.tags + [x.name])
         ]
 
+    def filter_by_tags(self, tags: Sequence[str]) -> None:
```

Comment:
Any particular reason for using `Sequence[str]` here? Everywhere else it's a `list[str]`, if we can be more specific and use `list` I don't see the benefit of using the more generic `Sequence` IMO

Reply:
@FollowTheProcess I decided to use the same signature as [`Manifest.filter_by_python_interpreter`](https://github.com/wntrblm/nox/blob/d1bdf0940eda7213dc2bd6407cf7107dfc986672/nox/manifest.py#L157), and it also made sense to follow the [Robustness principle](https://en.wikipedia.org/wiki/Robustness_principle). Wdyt?

Reply:
I'd say just stick with a `list[str]` personally, it's a `list[str]` everywhere else it shows up it makes more sense to keep it consistent IMO rather than have the one off `Sequence` here. As for the robustness thing, that doesn't matter so much here as it's internal stuff we can change the signature of without breaking people

Reply:
@FollowTheProcess Sounds good to me. I've pushed the change.